simulation/simulation_interface/proto/simulation_api_schema.proto

```diff
@@ -154,18 +176,16 @@ message DespawnEntityResponse {
  * Requests updating entity status.
  **/
 message UpdateEntityStatusRequest {
-  repeated traffic_simulator_msgs.EntityStatus status = 1;                 // List of updated entity status in traffic simulator.
-  traffic_simulator_msgs.VehicleCommand vehicle_command = 2;               // Autoware (Ego)'s vehicle command
-  traffic_simulator_msgs.EntityStatus ego_entity_status_before_update = 3; // Entity status of ego entity before running vehicle model
-  bool ego_entity_status_before_update_is_empty = 4;                       // If True,ego entity status before update is empty.
+  EntityStatus status = 1;                 // Updated entity status in traffic simulator.
+  bool npc_logic_started = 2;              // Npc logic started flag
 }
 
 /**
  * Response of updating entity status.
  **/
 message UpdateEntityStatusResponse {
   Result result = 1;                       // Result of [UpdateEntityStatusRequest](#UpdateEntityStatusRequest)
-  repeated UpdatedEntityStatus status = 2; // List of updated entity status in sensor/dynamics simulator.
+  UpdatedEntityStatus status = 2;          // Updated entity status in sensor/dynamics simulator
```

Comment:
Why do you remove `repeated`.
All entities can be overwrite it's status in `simple_sensor_simulator`.

Reply:
Thanks, we now confirming why the schema should be changed.

I do not feel that this implementation is a big problem at this time, but I will track the details of the discussion as it may cause performance degradation and data inconsistencies in the future.